release/release.go

```diff
@@ -151,6 +151,38 @@ func CheckUpstreamRelease(ctx context.Context, client *github.Client, org, repo
 	return releases, nil
 }
 
+func KubernetesGoVersion(ctx context.Context, client *github.Client, version string) (string, error) {
```

Comment:
Can you remove the debug logging from this function? The caller can choose if they want to log the error or not.